requirements.txt

```diff
@@ -5,4 +5,5 @@ ruff
 aiohttp
 ifaddr
 pycryptodome
-build
\ No newline at end of file
+build
+defusedxml
```

Comment:
It also needs to be added to `setup.py`.

Reply:
Oh indeed, I'm too used to poetry ;-)